tests/forms_tests/tests/test_utils.py

```diff
@@ -162,3 +163,50 @@ def test_error_list_html_safe(self):
         e = ErrorList(['Invalid username.'])
         self.assertTrue(hasattr(ErrorList, '__html__'))
         self.assertEqual(str(e), e.__html__())
+
+    def test_error_dict_is_dict(self):
+        self.assertIsInstance(ErrorDict(), dict)
+
+    def test_error_dict_is_json_serializable(self):
+        inititems = ErrorDict([
+            ('__all__', ErrorList([
+                ValidationError('Sorry this form only works on leap days.',)
+            ])),
+            ('name', ErrorList([
+                ValidationError('This field is required.')
+            ])),
+        ])
+        e = ErrorDict(
+            inititems,
+            date=ErrorList([
+                ErrorDict({
+                    'day': ErrorList([
+                        ValidationError('Ensure this value is greater than or equal to 0.')
+                    ]),
+                    'month': ErrorList([
+                        ValidationError('Ensure this value is greater than or equal to 0.')
+                    ]),
+                    'year': ErrorList([
+                        ValidationError('Ensure this value is greater than or equal to 0.')
+                    ]),
+                })]
+            ),
+        )
+        e['renderer'] = ErrorList([
+            ValidationError('Select a valid choice. That choice is not one of the available choices.')
+        ])
+        self.assertEqual(
```

Comment:
This is a much better test that visibly demonstrates things _should_ work again, nice. :)

I don't know the stylistic preferences @felixxm _et al_ might have around using `assertEqual` vs. `assertJSONEqual` here, so I'll leave this as a note for highlighting.

Reply:
I wasn't aware of `assertJSONEqual`, I'll update the PR to use that instead